styles.css

```diff
@@ -198,3 +200,532 @@ a {
   color: unset;
   cursor: pointer;
 }
+
+#action-btn-cont {
+  position: absolute;
+  top: 665px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 327px;
+  height: 48px;
+}
+
+#action-btn {
+  width: 40px;
+  height: 40px;
+  background-image: url("images/Disabled.png");
+  background-size: cover;
+}
+
+/* #endregion */
+
+/* #region Works Section */
+
+#works {
+  display: flex;
+  position: absolute;
+  top: 708px;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 3283px;
+}
+
+#works-head-cont {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 64px;
+  width: 327px;
+  height: 125px;
+}
+
+#works-header {
+  width: 327px;
+  height: 97px;
+  overflow-wrap: break-word;
+}
+
+.section-header {
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  text-align: center;
+}
+
+h2 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+#indicator {
+  width: 48px;
+  height: 4px;
+  background-image: url("images/Indicator.png");
+  background-size: cover;
+}
+
+#cards-cont {
+  margin-top: 58px;
+  display: grid;
+  grid-template-rows: repeat(6, 474px);
+  width: 327px;
+  height: 2972px;
+  grid-gap: 24px;
+}
+
+.card {
+  width: 327px;
+  height: 474px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+#card1 {
+  grid-row: 1/2;
+}
+
+#card2 {
+  grid-row: 2/3;
+}
+
+#card3 {
+  grid-row: 3/4;
+}
+
+#card4 {
+  grid-row: 4/5;
+}
+
+#card5 {
+  grid-row: 5/6;
+}
+
+#card6 {
+  grid-row: 6/7;
+}
+
+.project-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 222px;
+  margin-top: 252px;
+  background-color: #fff;
+  border-radius: 8px;
+  gap: 16px;
+}
+
+h3 {
+  margin-block-end: 0;
+  margin-block-start: 0;
+  font-weight: unset;
+}
+
+.project-name {
+  width: 295px;
+  height: 57px;
+  overflow-wrap: break-word;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  text-align: center;
+  margin-top: 16px;
+}
+
+.pr-tags-ul {
+  display: flex;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  gap: 8px;
+}
+
+.pr-tag {
+  list-style-type: none;
+  background-color: #ebf0ee;
+  height: auto;
+  width: auto;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  padding: 8px 12px;
+  border-radius: 4px;
+}
+
+#btn-cont {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 305px;
+  height: 69px;
+}
+
+#pr-detail-btn {
+  width: 122px;
+  height: 48px;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+}
```

Comment:
- [ ] Kindly fix the button color. Implement the button interactions (enable, hover, etc.) also to match the given design.
- ![image](https://github.com/s2jack/create-portfolio-website1/assets/60083437/76815ad5-2d5d-4b31-aea3-0dd35e2ee555)